Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md

```diff
@@ -0,0 +1,8 @@
+# QR Code Generator
+This is a simple code that can generate qr codes for various purposes. Few use cases are explained below.
+
+## Event Invitations
+Ever had to organize hackathons/workshops? Some  hackathons are 24 hours long, the organizers arrange for food goodie etc..
```

Comment:
A small typo, you have included, double full stops

Reply:
i thought sentences ending with etc have double full stops 
as in 1 for etc and one normal full stop 
no worries i'll change it

Reply:
@adityaSINGHrox Oh, I use Grammarly for checking my READMEs, if this seems good to you and is accepted, I have no problem with it

Reply:
> @adityaSINGHrox Oh, I use Grammarly for checking my READMEs, if this seems good to you and is accepted, I have no problem with it

oh if that's the case i'll do it